resources/assets/less/bem/bootstrap/modal.less

```diff
@@ -0,0 +1,50 @@
+/**
+*    Copyright 2015 ppy Pty. Ltd.
+*
+*    This file is part of osu!web. osu!web is distributed with the hope of
+*    attracting more community contributions to the core ecosystem of osu!.
+*
+*    osu!web is free software: you can redistribute it and/or modify
+*    it under the terms of the Affero GNU General Public License as published by
+*    the Free Software Foundation, either version 3 of the License, or
+*    (at your option) any later version.
+*
+*    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+*    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*    See the GNU Affero General Public License for more details.
+*
+*    You should have received a copy of the GNU Affero General Public License
+*    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+
+.modal {
+	// FIXME: Chrome sometimes fail painting the modal box without this
+	backface-visibility: hidden;
+
+	// Elements
+	&__dialog {
+		width: 430px;
+		color: @text-color;
+	}
+
+	// Modifiers
+}
+
+// Javascript-based hooks/styling
+.js-user-dropdown-modal { 
```

Comment:
Yeah, probably, although since we're already having a custom prefix for `js-`, which might be styled sometimes (like now), then the only other solution is creating new files, which I don't know is a good idea when you (maybe in the future) port the entire CSS to BEM-style.

Until this is decided, I think it's safe to simply keep it like this..